typescript/src/constants.ts

```diff
@@ -7,4 +7,4 @@ export const SOMBRA_URL =
 
 export const PORT = process.env.PORT || 8081;
 
-export const MEDIA_FOLDER = path.join(__dirname, '../..', 'media');
+export const MEDIA_FOLDER = path.join(__dirname, 'media');
```

Comment:
```suggestion
export const MEDIA_FOLDER = path.join(__dirname, '..', 'media');
```

Reply:
Are you sure? media is on `src` with the src code

Reply:
oh i was going off what you have here: https://github.com/transcend-io/examples/pull/41/files#diff-cf5fccf96a15ae53f23725827d9abcc112d035facf3e9920b175d5fba96f8726R8

i would not put the media folder in src because the location of the folder will change when you run `yarn tsc` keeping the folder at the same level as src and build will make it so that the relative path to the media folder is the same regardless of .ts or .js extension

Reply:
Oh that is what I had originally! Ok. I will move files up!